build-support/cherry_pick/make_pr.sh

```diff
@@ -16,16 +16,29 @@ fi
 
 PR_NUM=$1
 MILESTONE=$2
+BRANCH_NAME=$3
 
 PR_INFO=$(gh pr view "$PR_NUM" --json title,labels,reviews,body,author)
 
-TITLE=$(echo "$PR_INFO" | jq .title)
+TITLE=$(echo "$PR_INFO" | jq -r .title)
 AUTHOR=$(echo "$PR_INFO" | jq -r .author.login)
-CATEGORY_LABEL=$(echo "$PR_INFO" | jq '.labels[] | select(.name|test("category:.")).name')
-REVIEWERS="$(echo "$PR_INFO" | jq -r '.reviews[].author.login' | tr '\n' ' ') $AUTHOR"
+CATEGORY_LABEL=$(echo "$PR_INFO" | jq -r '.labels[] | select(.name|test("category:.")).name')
+REVIEWERS="$(echo "$PR_INFO" | jq -r '.reviews[].author.login' | tr '\n' ' ')$AUTHOR"
```

Comment:
Just confirming that this doesn't merge the author into the last review name? e.g. if `foo` and `bar` reviewed, and `baz` is the author, this comes out as `foo bar baz` rather than `foo barbaz`.

Also, drive-by change, but given `' '`s are swapped to `,` later, what do you think about doing that directly here?

```suggestion
REVIEWERS="$(echo "$PR_INFO" | jq -r '.reviews[].author.login' | tr '\n' ',')$AUTHOR"
```

Reply:
Yes, the old command had a trailing newline, which meant trailing space.